library/math/n_choose_k_mod.hpp

```diff
@@ -1,46 +1,44 @@
 /** @file */
 #pragma once
+const int N = 1001;
 /**
- * @code{.cpp}
- *     n_choose_k<1'000'000'007> nk(n); // to use `choose` with inputs strictly < n
- *     n_choose_k<1'000'003> nk(1'000'003); // to use `choose_lucas` with arbitrarily large inputs
- * @endcode
+ * nCk with arbitrarily large integers requires O(MOD) precalc
```

Comment:
O(n) precalc**

Reply:
also this comment should describe what the arrays contain, not what the file does

Reply:
I think we should just remove this comment, it's pretty self explanatory what the arrays hokd.

these are already ignored here https://github.com/lrvideckis/programming_team_code/blob/main/tests/.config/Doxyfile#L972